modules/appeals_api/db/migrate/20210412175709_add_upload_submission_to_evidence_submissions.rb

```diff
@@ -0,0 +1,16 @@
+class AddUploadSubmissionToEvidenceSubmissions < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  def change
+    safety_assured do
```

Comment:
TIL. What does this do?

Reply:
strong migrations recommended it. It's similar to Prism's old "run_unsafe_migrations" where we're indicating that we know the risks of the operations we're running and that they'll be fine to perform.

Since our table is empty, we're pretty safe ;)